index.js

```diff
@@ -200,6 +200,31 @@ module.exports = function(app) {
         })
     })
 
+    n2kCallback = (msg) => {
```

Comment:
You need to define n2kCallback. Add `var n2kCallback` above after `var registered = []`

Otherwise it will be global.

Reply:
Thanks for catching that. I just pushed it.